db/re/item_db.txt

```diff
@@ -4924,7 +4936,7 @@
 6923,Bright_Fire_Lights,Bright Fire Lights,3,10,,10,,,,,,,,,,,,,{},{},{}
 6924,Red_Eye,Red Eye,3,10,,10,,,,,,,,,,,,,{},{},{}
 6925,Letter_Of_Prisoner,Letter Of Prisoner,3,10,,0,,,,,,,,,,,,,{},{},{}
-6926,Rune-Midgard_History_Book,Rune-Midgarts History Book,3,10,,0,,,,,,,,,,,,,{},{},{}
+6926,Midgards_Histories,Rune-Midgards History Book,3,10,,0,,,,,,,,,,,,,{},{},{}
```

Comment:
```suggestion
6926,Rune-Midgarts_History_Book,Rune-Midgarts History Book,3,10,,0,,,,,,,,,,,,,{},{},{}
```

Reply:
Midgard = Continent
Rune-Midgarts = Country

Reply:
**First and foremost:**
Official aegis name is **_Midgards_Histories_** , no need to revert the aegis name.

**To the Midgard confusion:**
Ragnarok Online is based on _norse mythology_.
Midgart neither exists in _norse mythology_, nor does _Korean Ragnarok Online_ differentiate between a Rune-Midgard and Rune-Midgart **"kingdom"**.

They've been spelling it as 룬미드가츠 on every single source i've been ever looking at.
They call the worldmap 룬미드가츠
They call the kingdom 룬미드가츠
They call related items 룬미드가츠

I personaly think that Midgar**t** is the result of wrong translations, just like with tons of others things related to Ragnarok Online and the Kingdom was refering to the actual Rune-Midgard simply because the entire story started much smaller then where we currently are.

Here... this is from the official Korean Ragnarok Online database, both the world map 
Worldmap (Which **we** refer as Midgard)
![image](https://user-images.githubusercontent.com/12261601/73482553-3ddcdd80-439e-11ea-9257-2a30f79f2e7e.png)
Item (Which **we** refer as Midgart)
![image](https://user-images.githubusercontent.com/12261601/73482570-43d2be80-439e-11ea-957e-2d736140ea4e.png)
Their Rune-Midgard database search
![image](https://user-images.githubusercontent.com/12261601/73482827-ba6fbc00-439e-11ea-9a43-93c4132b7aae.png)

It's just all the same.
I know we have a wiki that refers to Midgart as the *Kingdom*, but is that really a reliable source? It is most likely based on the tv and comic series which may have translation errors as well.

Their website source folders refer to midgart when linking to the Rune-Midgard Worldmap...
![image](https://user-images.githubusercontent.com/12261601/73483401-a7112080-439f-11ea-8c17-a0bfb4bf3d8a.png)

So how reliable is Midgar**t** really?

But i have an open ear if you anyone can come up with a proof that kRO actualy clearly refers to different names inbetween the kingdom and the actual official Rune-Midgard.

Really just my opinion, not trying to fight over this.

Reply:
Official aegis name is reasonable.

First the game ragnarok online is based on the manga(manhwa) written by Lee Myung-jin which was release somewhere in the 90's. Well, the general idea came from norse though.

> They've been spelling it as 룬미드가츠 on every single source i've been ever looking at.
They call the worldmap 룬미드가츠
They call the kingdom 룬미드가츠
They call related items 룬미드가츠
While 룬 stand for Rune and 미드가츠 for Midgard.

룬미드가츠 reads as Rune-Midgarts

The first photo you've sent is reads as Prontera, the capital of Rune-Midgarts.

Regarding in the map, it's written as orbis of midgard because it includes the 3 country/state
The so called Rune-Midgarts, Arunafeltz , and Schwarzwald.
Try pointing your map in Yuno which is the capital of Schwarzwald/Arunafeltz (not sure which one) for a quick confirmation that the world map is not rune-midgarts as a whole.

Reply:
미드가츠 = Midgar**d**s
It refers to the Midgard everyone knows and love in every single reference on Korean Ragnarok Online. Doesn't matter if it's the kingdom, or the world or anything else.

All items that we refer to MIDGART AND MIDGARD, in the Korean Ragnarok Online Database always reads as 미드가츠
There's no difference because Midgart doesn't exist.

So you're saying just because it "reads as" Midgar**t** it must exist?
Based on what?

If that's the case, then everything in the entire would would be called Midgar**t** instead of Midgar**d**. Which doesn't make sense because Midgard is the main world in the norse mythology and in Korean Ragnarok Online.

**I repeat:**
Please show evidence that Korean Ragnarok Online ever referes to anything else other than 룬미드가츠 when comparing the kingdom name to the world name (in actual korean).

I really can't find anything.

Reply:
> All items that we refer to MIDGART AND MIDGARD, in the Korean Ragnarok Online Database always reads as 미드가츠
There's no difference because Midgart doesn't exist.

Midgard Sample :
Item ID : 6242 - 미드가르드 주화 ( Midgard Coin ) - [kRO Link](http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=7&ID=6242)
Item ID : 11012 - 미드가르드 원정대의 보고서 ( Midgard Expedition Report ) - [kRO Link](http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=7&ID=11012)


Rune Midgarts Sample :
Item ID : 6926 - 룬-미드가츠 역사서 ( Rune Midgarts History ) - [kRO Link](http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=7&ID=6926)

Reply:
Thanks for showing me actual Midgard versions.
However let me show you this:

Item Description of Midgard Coin says:
**룬 미드가르드 왕국이** 발행한 24K 순금으로 만든 주화.
Issues by the Rune Midgar**d** Kingdom.

Lost in translation as usual. ;)